docs/web/docs/3-reference/3-schemas/okta_group_template.mdx

```diff
@@ -67,7 +78,9 @@ configurations for other models used in IAMbic.
   - **`deleted`** *(boolean)*: Denotes whether the resource has been removed from AWS.Upon being set to true, the resource will be deleted the next time iambic is ran. Default: `false`.
   - **`name`** *(string)*: Name of the group.
   - **`group_id`** *(string)*: Unique Group ID for the group. Usually it's {idp-name}-{name}. Default: `""`.
+  - **`file_path`** *(string)*: File path of the group. Default: `""`.
```

Comment:
this is kinda bad because it goes to the reference docs. we need to allocate field storage for file_path, but do not have anyone trying to set it. same for metadata_commented_dict, metadata_iambic_fields. one can make the argument metadata_xyz probably discourage people to set it, file_path maybe too tempting for users to set it inside the template (which we don't want)